docker/Dockerfile.runner

```diff
@@ -195,3 +195,76 @@ rm -rf /usr/lib/python2.7 && rm -rf /usr/lib/python3.6
 EOT
 
 VOLUME ["/volumes", "/snapshot"]
+
+# Micromamba-base
+# ========================
+FROM base as micromamba-base
+
+WORKDIR /workspace
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    NVIDIA_VISIBLE_DEVICES=all \
+    PATH=/micromamba/bin:$PATH \
+    MAMBA_ROOT_PREFIX=/micromamba \
+    MAMBA_EXE=/bin/micromamba \
+    SHELL=/bin/bash
+
+COPY ./pkg/abstractions/image/base_requirements.txt /workspace/requirements.txt
+
+SHELL ["/bin/bash", "-c"]
+RUN <<EOT
+set -eux
+
+VERSION="2.0.2"
+ARCH=$(uname -m)
+if [ "$ARCH" = "x86_64" ]; then
+    MAMBA_ARCH="64"
+elif [ "$ARCH" = "aarch64" ]; then
+    MAMBA_ARCH="aarch64"
+else
+    echo "Unsupported architecture: $ARCH"
+    exit 1
+fi
+
+curl -L "https://micro.mamba.pm/api/micromamba/linux-${MAMBA_ARCH}/${VERSION}" \
+| tar -xj -C "/" "bin/micromamba"
+
+mkdir -p /micromamba
+
+sh -c 'echo "#!/bin/sh\nexec micromamba run -n beta9 python3 \"\$@\""' > /micromamba/mpy
+chmod +x /micromamba/mpy
+
+apt-get clean -y
+apt-get autoremove -y
+rm -rf /var/lib/apt/lists/*
+rm -rf /usr/share/doc
+rm -rf /usr/share/man
+rm -rf /usr/share/locale
+rm -rf /root/.cache/*
+rm -rf /tmp/*
+EOT
+
+
+VOLUME ["/volumes", "/snapshot"]
+
+# Micromamba 3.10
```

Comment:
Nitpick probably, but is "3.10" is the python version in this? Can we remove that? Just better to focus on maintaining the build arg and not the comment.

Reply:
Oh yeah I should remove that